pycytominer/cyto_utils/sqlite/meta.py

```diff
@@ -0,0 +1,150 @@
+"""
```

Comment:
I think I've reviewed this code already? Oh, is it being moved over here from a different file?

Reply:
Apologies - yes, we've reviewed this code, it is moved from `cyto_utils/sqlite.py` and I mislabeled the commit change for this file (add vs rename/move).